go.mod

```diff
@@ -0,0 +1,57 @@
+module github.com/honeydipper/honeydipper
+
+go 1.12
+
+require (
+	cloud.google.com/go v0.31.0
+	github.com/DataDog/datadog-go v0.0.0-20180330214955-e67964b4021a
+	github.com/Masterminds/semver v1.4.2 // indirect
+	github.com/Masterminds/sprig v2.16.0+incompatible
+	github.com/aokoli/goutils v1.0.1 // indirect
+	github.com/emirpasic/gods v1.12.0 // indirect
+	github.com/ghodss/yaml v1.0.0
+	github.com/go-errors/errors v1.0.1
+	github.com/go-redis/redis v6.14.2+incompatible
+	github.com/gogo/protobuf v0.0.0-20170330071051-c0656edd0d9e // indirect
+	github.com/google/btree v1.0.0 // indirect
+	github.com/google/gofuzz v0.0.0-20161122191042-44d81051d367 // indirect
+	github.com/google/uuid v1.1.0 // indirect
+	github.com/googleapis/gax-go v2.0.0+incompatible // indirect
+	github.com/googleapis/gnostic v0.0.0-20170729233727-0c5108395e2d // indirect
+	github.com/gregjones/httpcache v0.0.0-20170728041850-787624de3eb7 // indirect
+	github.com/h2non/gock v1.0.14
+	github.com/honeydipper/honeydipper v0.0.0-20190807213123-0eabf175214b
+	github.com/huandu/xstrings v1.2.0 // indirect
+	github.com/imdario/mergo v0.3.7
+	github.com/json-iterator/go v0.0.0-20180612202835-f2b4162afba3 // indirect
+	github.com/logrusorgru/aurora v0.0.0-20181002194514-a7b3b318ed4e
+	github.com/mitchellh/mapstructure v1.1.2
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 // indirect
+	github.com/onsi/ginkgo v1.9.0 // indirect
+	github.com/onsi/gomega v1.6.0 // indirect
+	github.com/op/go-logging v0.0.0-20160211212156-b2cb9fa56473
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/spf13/pflag v1.0.3 // indirect
+	github.com/src-d/gcfg v1.4.0 // indirect
+	github.com/stretchr/testify v1.2.2
+	go.opencensus.io v0.18.0 // indirect
+	golang.org/x/crypto v0.0.0-20181030102418-4d3f4d9ffa16
+	golang.org/x/net v0.0.0-20181101160248-e11730110bbd // indirect
+	golang.org/x/oauth2 v0.0.0-20181101160152-c453e0c75759
+	golang.org/x/sys v0.0.0-20181031143558-9b800f95dbbc // indirect
+	golang.org/x/time v0.0.0-20161028155119-f51c12702a4d // indirect
+	google.golang.org/api v0.0.0-20181101164933-799fe93f8274
+	google.golang.org/appengine v1.3.0 // indirect
+	google.golang.org/genproto v0.0.0-20181101192439-c830210a61df
+	google.golang.org/grpc v1.16.0 // indirect
+	gopkg.in/inf.v0 v0.9.0 // indirect
+	gopkg.in/src-d/go-billy.v4 v4.3.0 // indirect
+	gopkg.in/src-d/go-git.v4 v4.7.1
+	gopkg.in/yaml.v2 v2.2.2 // indirect
+	k8s.io/api v0.0.0-20181004124137-fd83cbc87e76
+	k8s.io/apimachinery v0.0.0-20180913025736-6dd46049f395
+	k8s.io/client-go v9.0.0+incompatible
+)
+
+replace github.com/h2non/gock => gopkg.in/h2non/gock.v1 v1.0.14
```

Comment:
I think this is needed because of inconsistent imports; however, I couldn't find them. Would be good if we can find a better fix for this.